mod/redir.php

```diff
@@ -80,7 +80,7 @@ function redir_init(App $a) {
 		}
 
 		// Doing remote auth with dfrn.
-		if (local_user() && (!empty($contact['dfrn-id']) || !empty($contact['issued-id']))) {
+		if (local_user() && (!empty($contact['dfrn-id']) || !empty($contact['issued-id'])) && empty($contact['pending'])) {
```

Comment:
Why the check for "empty"?

Reply:
Because it is used for the other fields we also know exist?

Reply:
Because it checks if key doesn't exists or its value is 0 or false.
Is there something wrong to use empty here?

Reply:
it is okay, but I'm unsure if we only should use "empty" when we are unsure if that variable exists at all. @MrPetovan is more into the PHP coding standards to tell us about this.

Reply:
Since we aren't using classes with declared members, we can never be sure that a specific key exists in an array. Better use `empty` whenever we're dealing with arrays in general.